### DISCUSSION TITLE
Documentation Websites are Down

### DISCUSSION THREAD

**Discussion by code-poel** [roach-php/core#16] [General]
Any plan on getting documentation incorporated into the repository?  The documentation websites seem to be down.

Thanks!

**Comment by ksassnowski** [comment]
Wait what? It’s working for me https://roach-php.dev

The documentation has its own repository over here https://github.com/roach-php/roach-php.dev

**Comment by code-poel** [comment]
You're 100% right.  Checked on another computer and it's working which lead me to realize that my development machine's .dev domains are over-ridden for local development!  False alarm and thank you for the documentation link!
